examples/co2brinepvt.cpp

```diff
@@ -52,26 +52,41 @@ double densityBrine(const BrinePvt& brinePvt, const double p, const double T, co
 
 int main(int argc, char **argv)
 {
-    if (argc < 5) {
+
+    bool help = false;
+    for (int i = 1; i < argc; ++i) {
+        std::string tmp = argv[i];
+        help = help || (tmp  == "--h") || (tmp  == "--help");
+    }
+
+    if (argc < 5 || help) {
+        std::cout << "co2brinepvt computes PVT properties of a brine/co2 system " << std::endl;
+        std::cout << "for a given phase (oil or brine), pressure, temperature, salinity(optional), rs(optional) " << std::endl;
+        std::cout << "The properties support are: density, the inverse phase formation volume factor (invB), vicosity, " << std::endl;
+        std::cout << "saturated dissolution factor (rsSat) " << std::endl;
+        std::cout << "See CO2STORE in the OPM manual for more details." << std::endl;
+        std::cout << "USAGE:" << std::endl;
```

Comment:
Thanks.
Maybe we should put usage to one line and at the top?

Everywhere else we seem to use:
```
Usage: co2brinepvt <prop> <phase> <p> <T> <salinity> <rs>
... explanation
```

Reply:
updated